### PR TITLE
Add careless option to Patrol Area module

### LINKED ISSUE #339: Patrol module should have an option for careless
**Is your feature request related to a problem?**
When placing down civilians you want them not to react to players or other AI. So you would put them into careless and make them patrol. But there is no option in the patrol module so you have to do it separately and that is annoying.

**Solution you'd like:**
In the Patrol module add a new entry in the toolbox for `Careless`.

**Alternatives you've considered:**
N/A

**Additional context:**
N/A

### PR DESCRIPTION
**Pull request by SDTheCyanWyan**
**When merged this pull request will:**
- Add a `Careless` option to the Toolbox
- Will close #339

### CODE REVIEW

File: addons/modules/functions/fnc_modulePatrolArea.sqf [hunk 1]
```diff
@@ -51,7 +52,7 @@ if !(alive _unit) exitWith {
         private _speedMode = ["LIMITED", "NORMAL"] select (_behaviour > 1);
```
Comment (mharis001) [inline addons/modules/functions/fnc_modulePatrolArea.sqf]: Should also be changed, not sure what speed mode makes most sense for careless behaviour.

Comment (CreepPork) [reply addons/modules/functions/fnc_modulePatrolArea.sqf]: I would assume limited for the most part.

Comment (SDTheCyanWyan) [reply addons/modules/functions/fnc_modulePatrolArea.sqf]: Careless behavior is accompanied by Limited speed. Though maybe also having Relaxed be accompanied by Limited speed be a good idea

Comment (mharis001) [reply addons/modules/functions/fnc_modulePatrolArea.sqf]: Yes, change it to `_behaviour > 2`.

**Pull request merged**